build.sbt

```diff
@@ -1,4 +1,10 @@
-libraryDependencies ++= Seq("com.typesafe" % "config" % "1.2.0")
+resolvers += "Sonatype-Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
+
+libraryDependencies ++= Seq(
+  "com.typesafe" % "config" % "1.2.0",
+  "info.cukes" % "cucumber-scala_2.10" % "1.2.3" % "test",
```

Comment:
It's more standard practice to use a `%%` instead of plugging in `-scala_2.10`. That way, you don't need to update these by hand when the scala version changes.